src/se/vidstige/jadb/managers/PropertyManager.java

```diff
@@ -23,8 +23,15 @@ public PropertyManager(JadbDevice device) {
     }
 
     public Map<String, String> getprop() throws IOException, JadbException {
-        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(device.executeShell("getprop")));
-        return parseProp(bufferedReader);
+        BufferedReader bufferedReader = null;
+        try {
```

Comment:
Done in d3ef914. I keep forgetting the whole `try-with-resources` feature in Java.